caravel/models.py

```diff
@@ -23,7 +23,7 @@
 from flask_appbuilder import Model
 from flask_appbuilder.models.mixins import AuditMixin
 from flask_appbuilder.models.decorators import renders
-from flask_babelpkg import gettext as _
+from flask_babelpkg import lazy_gettext as _
```

Comment:
Wait that means you made everything else that didn't need to be lazy use lazy_gettext